docs/api-ref/rest-api/index.md

```diff
@@ -12,9 +15,117 @@ The [Prefect REST API](/api-ref/rest-api/) is used for communicating data from c
 
 Prefect Cloud and a locally hosted Prefect server each provide a REST API.
 
-- Interactive Prefect Cloud REST API documentation is available at <a href="https://app.prefect.cloud/api/docs" target="_blank">https://app.prefect.cloud/api/docs</a>.
-- Interactive REST API documentation for a locally hosted open-source Prefect server is available at <a href="http://localhost:4200/docs" target="_blank">http://localhost:4200/docs</a> or the `/docs` endpoint of the [`PREFECT_API_URL`](/concepts/settings/#prefect_api_url) you have configured to access the server. You must have the server running with `prefect server start` to access the interactive documentation.
-- The REST API documentation for a locally hosted open-source Prefect server is also available in the [Prefect REST API Reference](/api-ref/rest-api-reference/).
+- Prefect Cloud:
+    - [Interactive Prefect Cloud REST API documentation](https://app.prefect.cloud/api/docs).
+    - [Finding your Prefect Cloud details](#finding-your-prefect-cloud-details)
+- Locally hosted open-source Prefect server:
+    - Interactive REST API documentation for a locally hosted open-source Prefect server is available at http://localhost:4200/docs or the `/docs` endpoint of the [PREFECT_API_URL](/concepts/settings/#prefect_api_url) you have configured to access the server. You must have the server running with `prefect server start` to access the interactive documentation.
```

Comment:
```suggestion
    - Interactive REST API documentation for a locally hosted open-source Prefect server is available at `http://localhost:4200/docs` or the `/docs` endpoint of the [PREFECT_API_URL](/concepts/settings/#prefect_api_url) you have configured to access the server. You must have the server running with `prefect server start` to access the interactive documentation.
```

it might make sense to at least add dashes so it's not just a bare link

Reply:
similar to here:
https://deploy-preview-9661--prefect-docs-preview.netlify.app/concepts/infrastructure/?h=localhost#dockercontainer

Reply:
Also maybe this documentation could be the second bullet point